src/pages/asset.tsx

```diff
@@ -58,85 +63,64 @@ const Asset: NextPage = () => {
       {hasData ? (
         <>
           <Divider />
-          <Section
-            width={SectionSize.FULL}
-            sx={{
-              display: 'flex',
-              flexDirection: 'row',
-              marginBottom: '12px',
-              justifyContent: 'space-between',
-              alignItems: 'center',
-              background: theme.palette.background.paper,
-            }}
-          >
-            <Box
+          <Row>
+            <Section
+              width={SectionSize.FULL}
               sx={{
                 display: 'flex',
-                justifyContent: 'flex-start',
-              }}
-            >
-              <Typography sx={{ padding: '6px' }}>Oracle Price</Typography>
-              <Button
-                sx={{
-                  alignItems: 'center',
-                  padding: 0,
-                  cursor: 'pointer',
-                  minWidth: '0px',
-                  color: theme.palette.text.primary,
-                }}
-                onClick={() =>
-                  window.open(
-                    `${process.env.NEXT_PUBLIC_STELLAR_EXPERT_URL}/contract/${pool.metadata.oracle}`,
-                    '_blank'
-                  )
-                }
-              >
-                <OpenInNewIcon fontSize="inherit" />
-              </Button>
-            </Box>
-
-            <Typography sx={{ padding: '6px' }}>
-              {`$${poolOracle.getPriceFloat(reserve.assetId)?.toFixed(2) ?? ''}`}
-            </Typography>
-          </Section>
-          <Section width={SectionSize.FULL} sx={{ padding: '6px', display: 'flex' }}>
-            <Row
-              sx={{
-                padding: '6px',
-                margin: '6px',
-                borderRadius: '5px',
-                background: theme.palette.background.default,
-                alignItems: 'center',
-              }}
-            >
-              <Typography sx={{ padding: '6px' }}>Reserve Index</Typography>
-              <Typography sx={{ padding: '6px' }}>{reserve.config.index}</Typography>
-            </Row>
-            <Row
-              sx={{
-                padding: '6px',
-                margin: '6px',
-                borderRadius: '5px',
-                background: theme.palette.background.default,
+                flexDirection: 'row',
+                justifyContent: 'space-between',
                 alignItems: 'center',
+                background: theme.palette.background.paper,
               }}
             >
-              <Typography sx={{ padding: '6px' }}>Supply Emission Index</Typography>
-              <Typography sx={{ padding: '6px' }}>{reserve.getBTokenEmissionIndex()}</Typography>
-            </Row>
-            <Row
+              <Typography sx={{ padding: '6px' }}>Status</Typography>
+              <AssetStatusBox titleColor="inherit" status={assetStatus} />
```

Comment:
can these be rendered on the same row? It looks odd taking up a whole row for just each entry.

Check that this works on mobile, otherwise we can stack them as is if page is mobile